README.md

```diff
@@ -14,66 +14,45 @@ Takeaway Challenge
 
  ```
 
-Instructions
+Approach
 -------
+I approached this challenge by creating five classes:
+- A Menu class, which stores the menu
+- An Order class, which creates a new order, sends and order to be printer and confirm orders. 
+- A Calculator class, which calculates the total of an order
+- A Printer class, which prints out the menu and the bill
+- A Text class, which sends an SMS to confirm an order
+
+How to use it
+-----
+To see the menu:
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+In irb:
+require "./lib/menu"
 
-Task
------
+menu = Menu.new
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+menu.print 
 
-```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+To order:
+--------
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
+In irb:
```

Comment:
Might want to find out more about formatting in markdown, so you can clearly indicate ```code snippets``` using backticks.